x/staking/keeper/keeper_test.go

```diff
@@ -396,6 +404,35 @@ func (s *KeeperTestSuite) TestValidatorsMigrationToColls() {
 	s.Require().NoError(err)
 }
 
+func (s *KeeperTestSuite) TestRedelegationQueueMigrationToColls() {
+	s.SetupTest()
+
+	err := testutil.DiffCollectionsMigration(
+		s.ctx,
+		s.key,
+		100,
+		func(i int64) {
+			date := time.Date(2023, 8, 21, 14, 33, 1, 0, &time.Location{})
+			s.ctx.KVStore(s.key).Set(getRedelegationTimeKey(date), []byte{})
+		},
+		"035e246f9d0bf0aa3dfeb88acf1665684168256d8afb742ae065872d6334f6d6",
+	)
+	s.Require().NoError(err)
+
+	err = testutil.DiffCollectionsMigration(
+		s.ctx,
+		s.key,
+		100,
+		func(i int64) {
+			date := time.Date(2023, 8, 21, 14, 33, 1, 0, &time.Location{})
+			err := s.stakingKeeper.SetRedelegationQueueTimeSlice(s.ctx, date, nil)
+			s.Require().NoError(err)
+		},
+		"035e246f9d0bf0aa3dfeb88acf1665684168256d8afb742ae065872d6334f6d6",
+	)
+	s.Require().NoError(err)
```

Comment:
Let's use some more data here, otherwise we might be missing issues.
- Write something, not just `nil`
- Maybe use the int value to make the key (like using it as Unix for example), so we have many different keys